core_backend/tests/api/test_manage_content.py

```diff
@@ -44,6 +51,78 @@ def existing_content_id(
     )
 
 
+@pytest.fixture(scope="class")
+def temp_user(client: TestClient, db_session: Session) -> Generator[str, None, None]:
+    user3_db = UserDB(
+        username=TEMP_USERNAME,
+        hashed_password=get_password_salted_hash("temp_password"),
+        hashed_api_key=get_key_hash("temp_api_key"),
+        content_quota=TEMP_USER_CONTENT_QUOTA,
```

Comment:
If we do leave it nullable, we should test what happens when `content_quota` is null